sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -1277,7 +1280,7 @@ def get_module_fw_mgmt_feature(self, verbose = False):
 
         else:
             txt += 'Reply payload check code error\n'
-            return {'status': False, 'info': txt, 'result': None}
+            return {'status': False, 'info': txt, 'feature': None}
```

Comment:
@xinyulin Can we log an error with the value of status here?

Reply:
This error will be logged with the following format in the latest commit
```python
        else:
            txt += 'Status or reply payload check code error\n'
            logger.info(txt)
            logger.info('Fail to get fw mgnt feature, cdb status: 0x{}\n'.format(status))
            return {'status': False, 'info': txt, 'feature': None}
``` 
The new commit has been tested with firmware download, run, and commit flows without any errors. Could you please review it to see if anything is missing.

Reply:
In my opinion, there is no need to log here. The method caller will make the judgment and log the result.
for example:
https://github.com/sonic-net/sonic-utilities/blob/f50587a1ff65bb489231bfe45cec805fb32dbf00/sfputil/main.py#L1526-L1531

Reply:
@rqx110 I think capturing the `status` and `rpl_chkcode` as part of logs will help in providing more datapoints to debug in case of an error.